app-starters-common/app-starters-file-common/src/main/java/org/springframework/cloud/stream/app/file/FileConsumerProperties.java

```diff
@@ -33,49 +33,49 @@
 	 * 'contents' - the contents as bytes.
 	 * Default is 'contents'
 	 */
-	private FileReadingMode fileReadingmode = FileReadingMode.contents;
+	private FileReadingMode mode = FileReadingMode.contents;
```

Comment:
@ericbottard , please, pay attention to this change. And see commit comments on the matter.